docs/USAGE.md

```diff
@@ -64,10 +64,10 @@ Use these environment variables to configure the tracing library:
 | `OTEL_TRACE_DOMAIN_NEUTRAL_INSTRUMENTATION` |  Sets whether to intercept method calls when the caller method is inside a domain-neutral assembly. This is recommended when instrumenting IIS applications. | `false` |
 | `OTEL_PROFILER_PROCESSES` | Sets the filename of executables the profiler can attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with comma, for example: `MyApp.exe,dotnet.exe` |  |
 | `OTEL_PROFILER_EXCLUDE_PROCESSES` | Sets the filename of executables the profiler cannot attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with comma, for example: `MyApp.exe,dotnet.exe` |  |
-| `OTEL_DOTNET_TRACER_LOAD_AT_STARTUP` | Defines whether the tracer is created by the auto instrumentation library or not. The default value is `true`. For applications with manual instrumentation in place and tracer being initialized in the app this should be changed to `false`. Note that if this is disabled, the application code will be in charge of setting up the exporters and any other desired instrumentation, e.g. AddAspNetInstrumentation (see [custom instrumentation section](#configure-custom-instrumentation)). | `true` | 
+| `OTEL_DOTNET_TRACER_LOAD_AT_STARTUP` | Defines whether the tracer is created by the auto instrumentation library or not. The default value is `true`. For applications with manual instrumentation in place and tracer being initialized in the app this should be changed to `false`. Note that if this is disabled, the application code will be in charge of setting up the exporters and any other desired instrumentation, e.g. AddAspNetInstrumentation (see [custom instrumentation section](#configure-custom-instrumentation)). | `true` |
 | `OTEL_DOTNET_TRACER_ADDITIONAL_SOURCES` | Comma separated list of additional `ActivitySource` names to be added to the tracer at the startup. |  |
 | `OTEL_DOTNET_TRACER_LEGACY_SOURCES` | Comma separated list of additional legacy source names to be added to the tracer at the startup. |  |
-
+| `OTEL_DOTNET_TRACER_PROVIDER_PLUGINS` | Colon (:) separated list of OTel SDK tracer plugins repesented by `System.Type.AssemblyQualifiedName`. See more in dedicated section. |
```

Comment:
nit: it is indeed a "tracer provider builder" plugin but I think it will be clear if we say that it is an instrumentation (also there is a typo).
```suggestion
| `OTEL_DOTNET_INSTRUMENTATION_PACKAGES` | Colon (:) separated list of OTel SDK instrumentation packages represented by `System.Type.AssemblyQualifiedName`. See more in dedicated section. |
```

Reply:
How about `OTEL_DOTNET_TRACER_INSTRUMENTATION_PLUGINS`? I am not sure about the `PACKAGES` suffix as for me a package is the same as the assembly.

Reply:
Renamed here https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/196/commits/0417fffe2fe1a72d6f4e71f9ac222a4db340326e